### PR TITLE
Add git/Github website event post

### PR DESCRIPTION
**Pull request by evykassirer**

### CODE REVIEW

File: content/W2015/W15-event-websitegitworkshop.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Title: Github/git and Website Workshop
+Date: 2015-03-1 12:00-17:00
+Category: Events
+Tags: workshop
+Slug: git-website-workshop
+Author: Evy Kassirer
+Summary: Come learn about web design and build your own website, hosted on Github.
+
+Interested in creating your own personal website? Women in Computer Science will teach you the nuts and bolts of web design and how to use git through GitHub! Join us for an afternoon of learning, coding, and snacking! Participants are expected to have basic knowledge of HTML and CSS.
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md]: Try to 80-char justify your lines.

Comment (evykassirer) [inline content/W2015/W15-event-websitegitworkshop.md]: should that be a separate commit? (edit: nope)

File: content/W2015/W15-event-websitegitworkshop.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+RSVP: There are limited spots available! Please fill out the following form by February 28th. We will let you know if you've been selected by March 3rd. http://goo.gl/forms/ybYZ4uL8Ld
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md]: To make a link in markdown, use the following syntax:

```
[link text](link here)
```

That link as written won't be clickable. I'd change this to

```
[fill out this form](http://goo.gl/forms/ybYZ4uL8Ld)
```

Though, this is going up after Feb. 28th, so I'm not sure there's much point in listing it?

File: content/W2015/W15-event-websitegitworkshop.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Workshop Details ##
+
++ **Who:** Cis and trans women and non-binary folks
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md:16]: I'd change this to "This event is women-only", and have the women-only bit be a clickable link to our FAQ which I will merge tonight.

File: content/W2015/W15-event-websitegitworkshop.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
++ **What:** Github/git and website workshop
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md]: I'd recommend you use proper capitalization (GitHub) and use teletype font for git: surrounding it with backticks `will make it render like`git`

File: content/W2015/W15-event-websitegitworkshop.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
++ **Where:** Details will be emailed to participants
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md:18]: I'd just change this to TBA

File: content/W2015/W15-event-websitegitworkshop.md (hunk 2 above)
Comment (evykassirer) [inline content/W2015/W15-event-websitegitworkshop.md]: Fair. I'll leave it out then

File: content/W2015/W15-event-websitegitworkshop.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
++ **When:** Saturday, March 7th, 12-5pm
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md]: You can use an n-dash here instead of a hyphen, by using the HTML escape code `&ndash;`

File: content/W2015/W15-event-websitegitworkshop.md (hunk 3 above)
Comment (evykassirer) [inline content/W2015/W15-event-websitegitworkshop.md:16]: I really don't like the idea of saying women-only when we mean to include people of non-binary genders (who aren't women at all). But maybe we should discuss that as a committee more

**Comment by ehashman** [comment]
Oh, one more thing, this should be a single commit. Use `git rebase -i HEAD~2` and follow the instructions to squash them together. (see README/the OpenHatch git documentation if you get stuck, or ask me)

File: content/W2015/W15-event-websitegitworkshop.md (hunk 5 above)
Comment (evykassirer) [inline content/W2015/W15-event-websitegitworkshop.md:18]: will that confuse participants who are trying to find where it is later, but forget that they have an email or something? That's why I decided not to put TBA, but I agree that TBA is cleaner

**Comment by evykassirer** [comment]
Will do! I wasn't sure if the rebase was for combining typo fixes or to make the whole thing one single commit. Now I know :)

File: content/W2015/W15-event-websitegitworkshop.md (hunk 6 above)
Comment (evykassirer) [inline content/W2015/W15-event-websitegitworkshop.md]: will do - thanks!

File: content/W2015/W15-event-websitegitworkshop.md (hunk 4 above)
Comment (evykassirer) [inline content/W2015/W15-event-websitegitworkshop.md]: sweet thanks :)

**Comment by evykassirer** [comment]
sorry I know I need to rebase them together but I kept going back and forth between rebasing and then git would say I needed to pull again? Anyways, will fix once it's close to being ready

File: content/W2015/W15-event-websitegitworkshop.md (hunk 5 above)
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md:18]: I don't think it will confuse people... TBA means "To be announced", which is exactly what you're saying, but more succinct.

File: content/W2015/W15-event-websitegitworkshop.md (hunk 3 above)
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md:16]: This is a short blurb section. I don't think we should really feel compelled to write out this full description everywhere we use it if we've defined it clearly and ensure it is very discoverable. In the end, we are "Women" in CS, so this problem kinda fundamentally affects our entire org. In the very least, this is consistent.

**Comment by ehashman** [comment]
Hey Evy, take a look at the [network tab](https://github.com/wics-uw/website/network) for a pictorial view of what your commit tree looks like. Somehow your three commits diverged and then you merged them together? I'm a bit confused as to how that happened. Anyways, you won't be able to rebase them with a merge commit on top. We should chat about how to fix that.

Once you rebase, you've rewritten your git tree, so if you are tracking a remote branch and you check your status, it will tell you your branches have diverged and there's different stuff remotely than there is locally. That's to be expected.

**Comment by evykassirer** [comment]
Wanna just chat about this tomorrow after our meeting? And can you ping
Yamna for the FAQ?

File: content/W2015/W15-event-websitegitworkshop.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+Summary: Come learn about web design and build your own website, hosted on Github.
+
+Interested in creating your own personal website? Women in Computer Science 
+will teach you the nuts and bolts of web design and how to use `git` through 
+GitHub! Join us for an afternoon of learning, coding, and snacking! Participants
+are expected to have basic knowledge of HTML and CSS.
+
+## Workshop Details ##
+
++ **Who:** Cis and trans women and non-binary folks
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md:16]: You can now change this to `"[Women-only]({filename}/pages/faq.md)` or something similar now that #46's been merged :)

File: content/W2015/W15-event-websitegitworkshop.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
++ **What:** Github/`git` and website workshop
```
Comment (ehashman) [inline content/W2015/W15-event-websitegitworkshop.md:17]: This instance of GitHub isn't capitalized properly, but I don't think you missed any others.

**Comment by ehashman** [comment]
I manually merged this; something really wonky happened and you had duplicate commits.

**Comment by annalorimer** [comment]
Argh weird.
